cmd/clusterctl/client/repository/repository_github.go

```diff
@@ -249,7 +258,12 @@ func NewGitHubRepository(ctx context.Context, providerConfig config.Provider, co
 		o(repo)
 	}
 
-	if token, err := configVariablesClient.Get(config.GitHubTokenVariable); err == nil {
+	// NOTE: tokens for private Github Enterprise domains must be of the format, github-token-{host}
```

Comment:
Is this a global definition or a definition from this PR?

Reply:
`github-token` is a global definition that's used to get to public github.com token. I introduced the `github-token-{host}` key in this PR to identify tokens for each github instance.

Reply:
Then we need to properly document this syntax so people know how to use it without reading code 👍

Reply:
@chrischdi I have added these changes to docs. See file: https://github.com/kubernetes-sigs/cluster-api/pull/9826/files#diff-4127eae8106dc3a0709a4131a0007cc3dd8b0393ce07916829b1b4394603aecfR32-R47

Reply:
Could you explain that "tokens for private Github Enterprise domains must be of the format, github-token-{host}" in the docs as well?